### PR TITLE
Update SearchableItem description

### PR DESCRIPTION
**Pull request by dblandin**
Add formatting for Article and Sale types. Fallback to the search result item's description, stripping all html tags.

### CODE REVIEW

File: src/schema/searchableItem.ts [hunk 1]
```diff
@@ -38,7 +41,22 @@ export const SearchableItem = new GraphQLObjectType<any, ResolverContext>({
     },
     description: {
       type: GraphQLString,
-      resolve: item => item.description,
+      resolve: ({ description, end_at, label, published_at, start_at }) => {
+        switch (label) {
+          case "Article":
+            return moment.utc(published_at).format("MMM Do, YYYY")
```
Comment (mzikherman) [inline src/schema/searchableItem.ts]: I thought the description for articles was this date, but only prepended to the actual description field? Now you're only returning the date, but dropping the description.

Comment (dblandin) [reply src/schema/searchableItem.ts]: Ahh, I lost some of the copy around the date here. I'll add it back in.

Comment (dblandin) [reply src/schema/searchableItem.ts]: Oh, and I didn't realize that we were actually getting descriptions from Elasticsearch, but you're totally right, we should join the two together.

**Comment by dblandin** [comment]
Going to go ahead and merge this in.

**Comment by mzikherman** [comment]
Sweet! @dblandin  Are there other types that require some custom descriptions? I'm thinking mainly of Shows, but possibly some others (gleaned from comparing the existing implementation in Force: https://github.com/artsy/force/blob/fa3ad6b5da839036a0af2aac2176270faff30d03/src/desktop/models/search_result.coffee#L89

**Comment by dblandin** [comment]
> Are there other types that require some custom descriptions?

Probably! I can follow-up and add those other formats in a separate PR :+1:

**Pull request merged**